src/styles/colors.scss

```diff
@@ -1,16 +1,62 @@
+// Grays -----------------------------------
 $shadow-color: #666;
 
-$medium-dark-blue: hsl(208, 59%, 49%);
-
 $lightest-gray: hsl(210, 13%, 95%);
 
 $lighter-gray: hsl(210, 12%, 93%);
 
 $light-gray: hsl(210, 12%, 80%);
 
+$light-brown-gray: rgba(27, 31, 35, 0.35);
+
 $medium-gray: hsl(210, 13%, 65%);
+
 $darker-gray: slategray;
 
-$light-brown-gray: rgba(27, 31, 35, 0.35);
 
+// Reds -------------------------------------
 $warning-red: crimson;
+
+$dark-red: darkred;
+
+$first-red: hsl(9, 90%, 88%);
+
+$second-red: hsl(-355, 96%, 83%);
+
+$third-red: hsl(0.61, 89%, 71%);
+
+$fourth-red: hsl(0.61, 89%, 71%);
+
+
+// Yellows ---------------------------------
+$mello-yello: hsl(55, 95%, 75%);
+
+
+// Greens ----------------------------------
+$save-green: hsl(150, 55%, 44%);
+
+$pale-green: hsl(208, 80%, 90%);
+
+
+// Blues -----------------------------------
+$light-dark-blue: hsl(208, 79%, 79%);
+
+$really-light-blue: hsl(212, 100%, 90%);
+
+$medium-dark-blue: hsl(208, 59%, 49%);
+
+
+// Submission_Detail Colors -----------------
+$unavailable-color: $lighter-gray;
+
+$all-correct-color: $really-light-blue;
+
+$all-correct-hover-color: $light-dark-blue;
+
+$none-correct-color: $third-red;
+
+$none-correct-hover-color: hsl(0.61, 85%, 60%);
```

Comment:
I would encourage you to use the SASS lighten and darken functions when we have to very closely related colors like this http://sass-lang.com/documentation/Sass/Script/Functions.html